docs/TypeConverters/index.md

```diff
@@ -0,0 +1,148 @@
+# Type Converters
+
+Provides a unified way of converting types of values to other types, as well as for accessing values and supporting custom serialzation.
+
+## Using Type Converters
+
+Accessing a type converter retrieves the same instance always. 
+
+```csharp
+class SomeClass
+{
+	string m_ID;
+
+	int GetID()
+	{
+		auto converter = CF_TypeConverter.Get(string);
+
+		converter.SetString(m_ID);
+
+		return converter.GetInt();
+	}
+
+	void SetID(int id)
+	{
+		auto converter = CF_TypeConverter.Get(string);
+
+		converter.SetInt(id);
+
+		m_ID = converter.GetString();
+	}
+}
+```
```

Comment:
Maybe explain the example with a short sentence like:

> This example shows how the type converter is used to represent an integer value as a string for internal storage

or something like that ...